homeassistant/components/sensor/bom.py

```diff
@@ -181,14 +178,49 @@ def __init__(self, hass, station_id):
         """Initialize the data object."""
         self._hass = hass
         self._zone_id, self._wmo_id = station_id.split('.')
-        self.data = None
+        self._data = None
         self._lastupdate = LAST_UPDATE
 
     def _build_url(self):
         url = _RESOURCE.format(self._zone_id, self._zone_id, self._wmo_id)
         _LOGGER.info("BOM URL %s", url)
         return url
 
+    @property
+    def latest_data(self):
+        """Return the latest data object."""
+        if self._data:
+            return self._data[0]
+        return None
+
+    def get_reading(self, condition):
+        """Return the value for the given condition.
+
+        BOM weather publishes condition readings for weather (and a few other
+        conditions) at intervals throughout the day. To avoid a `-` value in
+        the frontend for these conditions, we traverse the historical data
+        for the latest value that is not `-`.
+
+        Iterators are used in this method to avoid iterating needlessly
+        iterating through the entire BOM provided dataset
+        """
+        def should_include_value(value):
+            """Filter items from a set of condition readings."""
+            if condition == 'weather':
+                # Take the first non '-' reading for weather
+                return value is not None and value != '-'
+
+            # By default, take the first value regardless
+            return True
+
+        condition_readings = (entry[condition] for entry in self._data)
+        latest_reading = next(
+            (x for x in condition_readings if should_include_value(x)), None)
+        if latest_reading == '-':
+            # Show a unknown state if a '-' value is encountered.
```

Comment:
It's weird that `-` values in the state are handled differently for `weather` conditions than for other conditions. Shouldn't we just always take the first non `-` value?

Like this:

```python
condition_readings = (entry[condition] for entry in self._data)
return next((x for x in condition_readings if x != '-'), None)
```

Reply:
I think we can. I was under the impression that `weather` was some special case, but it looks like I can't see why this would negatively impact other conditions, like `cloud`. 

I had a look through the data paying attentions to `cloud` and `weather` to see if we could take this approach, and it looks like it'd be fine to do.

Also, it seems my previous assumption about `weather` being updated only every 3 hours is incorrect - there are some extra readings in there where it looks like the condition changed prematurely, so this is good reason to forego the update interval change.

```
{"local_date_time":"02/07:00am","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"02/06:30am","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"02/06:00am","weather":"Fine","cloud":"Mostly cloudy","cloud_base_m":2500,"cloud_oktas":7,"cloud_type_id":30,"cloud_type":"-"}
{"local_date_time":"02/05:30am","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"02/05:00am","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"02/04:30am","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"02/04:00am","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"02/03:30am","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"02/03:00am","weather":"Fine","cloud":"Mostly cloudy","cloud_base_m":2500,"cloud_oktas":6,"cloud_type_id":30,"cloud_type":"-"}
{"local_date_time":"02/02:30am","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"02/02:00am","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"02/01:30am","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"02/01:00am","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"02/12:30am","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"02/12:00am","weather":"Fine","cloud":"Mostly cloudy","cloud_base_m":2500,"cloud_oktas":7,"cloud_type_id":30,"cloud_type":"-"}
{"local_date_time":"01/11:30pm","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/11:00pm","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/10:30pm","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/10:00pm","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/09:30pm","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/09:00pm","weather":"Fine","cloud":"Partly cloudy","cloud_base_m":2500,"cloud_oktas":4,"cloud_type_id":30,"cloud_type":"-"}
{"local_date_time":"01/08:30pm","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/08:00pm","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/07:30pm","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/07:00pm","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/06:30pm","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/06:00pm","weather":"Fine","cloud":"Mostly clear","cloud_base_m":2500,"cloud_oktas":1,"cloud_type_id":30,"cloud_type":"-"}
{"local_date_time":"01/05:30pm","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/05:00pm","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/04:30pm","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/04:00pm","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/03:30pm","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/03:00pm","weather":"Fine","cloud":"Mostly clear","cloud_base_m":2500,"cloud_oktas":1,"cloud_type_id":30,"cloud_type":"-"}
{"local_date_time":"01/02:30pm","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/02:00pm","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/01:30pm","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/01:00pm","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/12:30pm","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/12:00pm","weather":"Fine","cloud":"Mostly clear","cloud_base_m":2500,"cloud_oktas":1,"cloud_type_id":30,"cloud_type":"-"}
{"local_date_time":"01/11:30am","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/11:00am","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/10:30am","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/10:00am","weather":"-","cloud":"Mostly clear","cloud_base_m":1020,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"01/09:30am","weather":"-","cloud":"Mostly clear","cloud_base_m":960,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"01/09:00am","weather":"Fine","cloud":"Mostly clear","cloud_base_m":600,"cloud_oktas":2,"cloud_type_id":31,"cloud_type":"-"}
{"local_date_time":"01/08:30am","weather":"-","cloud":"Mostly clear","cloud_base_m":750,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"01/08:00am","weather":"-","cloud":"Mostly clear","cloud_base_m":720,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"01/07:30am","weather":"-","cloud":"Mostly clear","cloud_base_m":660,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"01/07:00am","weather":"-","cloud":"Mostly clear","cloud_base_m":660,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"01/06:30am","weather":"-","cloud":"Mostly clear","cloud_base_m":660,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"01/06:00am","weather":"Fine","cloud":"Mostly clear","cloud_base_m":600,"cloud_oktas":1,"cloud_type_id":31,"cloud_type":"-"}
{"local_date_time":"01/05:30am","weather":"-","cloud":"Mostly clear","cloud_base_m":600,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"01/05:00am","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/04:30am","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/04:00am","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/03:30am","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/03:00am","weather":"Fine","cloud":"Clear","cloud_base_m":2500,"cloud_oktas":0,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/02:30am","weather":"-","cloud":"-","cloud_base_m":null,"cloud_oktas":null,"cloud_type_id":null,"cloud_type":"-"}
{"local_date_time":"01/02:00am","weather":"-","cloud":"Mostly clear","cloud_base_m":600,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"01/01:30am","weather":"-","cloud":"Mostly clear","cloud_base_m":600,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"01/01:00am","weather":"-","cloud":"Mostly clear","cloud_base_m":600,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"01/12:30am","weather":"-","cloud":"Mostly clear","cloud_base_m":600,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"01/12:00am","weather":"Fine","cloud":"Mostly clear","cloud_base_m":600,"cloud_oktas":1,"cloud_type_id":31,"cloud_type":"-"}
{"local_date_time":"30/11:30pm","weather":"-","cloud":"Mostly clear","cloud_base_m":600,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/11:00pm","weather":"-","cloud":"Mostly clear","cloud_base_m":600,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/10:30pm","weather":"-","cloud":"Mostly clear","cloud_base_m":600,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/10:00pm","weather":"-","cloud":"Mostly clear","cloud_base_m":600,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/09:30pm","weather":"-","cloud":"Mostly clear","cloud_base_m":600,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/09:00pm","weather":"Fine","cloud":"Mostly clear","cloud_base_m":600,"cloud_oktas":1,"cloud_type_id":31,"cloud_type":"-"}
{"local_date_time":"30/08:30pm","weather":"-","cloud":"Mostly clear","cloud_base_m":600,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/08:00pm","weather":"-","cloud":"Mostly clear","cloud_base_m":600,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/07:30pm","weather":"-","cloud":"Mostly clear","cloud_base_m":540,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/07:00pm","weather":"-","cloud":"Mostly clear","cloud_base_m":540,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/06:30pm","weather":"-","cloud":"Mostly clear","cloud_base_m":540,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/06:00pm","weather":"Distant precip.","cloud":"Mostly clear","cloud_base_m":300,"cloud_oktas":1,"cloud_type_id":38,"cloud_type":"-"}
{"local_date_time":"30/05:30pm","weather":"-","cloud":"Mostly clear","cloud_base_m":540,"cloud_oktas":1,"cloud_type_id":7,"cloud_type":"Stratus"}
{"local_date_time":"30/05:00pm","weather":"-","cloud":"Mostly clear","cloud_base_m":540,"cloud_oktas":1,"cloud_type_id":7,"cloud_type":"Stratus"}
{"local_date_time":"30/04:30pm","weather":"-","cloud":"Mostly clear","cloud_base_m":540,"cloud_oktas":1,"cloud_type_id":7,"cloud_type":"Stratus"}
{"local_date_time":"30/04:00pm","weather":"-","cloud":"Mostly clear","cloud_base_m":750,"cloud_oktas":2,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/03:30pm","weather":"-","cloud":"Mostly clear","cloud_base_m":540,"cloud_oktas":1,"cloud_type_id":7,"cloud_type":"Stratus"}
{"local_date_time":"30/03:00pm","weather":"Distant precip.","cloud":"Partly cloudy","cloud_base_m":600,"cloud_oktas":5,"cloud_type_id":38,"cloud_type":"-"}
{"local_date_time":"30/02:30pm","weather":"-","cloud":"Mostly clear","cloud_base_m":750,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/02:00pm","weather":"-","cloud":"Mostly clear","cloud_base_m":750,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/01:30pm","weather":"-","cloud":"Mostly clear","cloud_base_m":660,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/01:00pm","weather":"-","cloud":"Mostly clear","cloud_base_m":660,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/12:30pm","weather":"-","cloud":"Mostly clear","cloud_base_m":660,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/12:00pm","weather":"Distant precip.","cloud":"Partly cloudy","cloud_base_m":600,"cloud_oktas":5,"cloud_type_id":38,"cloud_type":"-"}
{"local_date_time":"30/11:30am","weather":"-","cloud":"Partly cloudy","cloud_base_m":810,"cloud_oktas":3,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/11:00am","weather":"-","cloud":"Partly cloudy","cloud_base_m":810,"cloud_oktas":3,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/10:30am","weather":"-","cloud":"Mostly clear","cloud_base_m":750,"cloud_oktas":2,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/10:00am","weather":"-","cloud":"Mostly clear","cloud_base_m":450,"cloud_oktas":1,"cloud_type_id":7,"cloud_type":"Stratus"}
{"local_date_time":"30/09:30am","weather":"-","cloud":"Partly cloudy","cloud_base_m":810,"cloud_oktas":3,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/09:00am","weather":"Distant precip.","cloud":"Partly cloudy","cloud_base_m":600,"cloud_oktas":4,"cloud_type_id":38,"cloud_type":"-"}
{"local_date_time":"30/08:30am","weather":"-","cloud":"Mostly clear","cloud_base_m":450,"cloud_oktas":1,"cloud_type_id":7,"cloud_type":"Stratus"}
{"local_date_time":"30/08:00am","weather":"-","cloud":"Mostly clear","cloud_base_m":450,"cloud_oktas":1,"cloud_type_id":7,"cloud_type":"Stratus"}
{"local_date_time":"30/07:30am","weather":"Showers","cloud":"Mostly clear","cloud_base_m":450,"cloud_oktas":1,"cloud_type_id":7,"cloud_type":"Stratus"}
{"local_date_time":"30/07:00am","weather":"-","cloud":"Mostly clear","cloud_base_m":540,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/06:30am","weather":"-","cloud":"Mostly clear","cloud_base_m":540,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/06:00am","weather":"Distant precip.","cloud":"Partly cloudy","cloud_base_m":300,"cloud_oktas":3,"cloud_type_id":38,"cloud_type":"-"}
{"local_date_time":"30/05:30am","weather":"-","cloud":"Mostly clear","cloud_base_m":540,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/05:00am","weather":"-","cloud":"Mostly clear","cloud_base_m":540,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/04:30am","weather":"-","cloud":"Mostly clear","cloud_base_m":600,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/04:00am","weather":"-","cloud":"Mostly clear","cloud_base_m":360,"cloud_oktas":1,"cloud_type_id":7,"cloud_type":"Stratus"}
{"local_date_time":"30/03:30am","weather":"-","cloud":"Mostly clear","cloud_base_m":720,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/03:00am","weather":"Recent precip.","cloud":"Partly cloudy","cloud_base_m":300,"cloud_oktas":5,"cloud_type_id":38,"cloud_type":"-"}
{"local_date_time":"30/02:30am","weather":"Showers","cloud":"Mostly clear","cloud_base_m":360,"cloud_oktas":1,"cloud_type_id":7,"cloud_type":"Stratus"}
{"local_date_time":"30/02:20am","weather":"Showers","cloud":"Mostly clear","cloud_base_m":360,"cloud_oktas":1,"cloud_type_id":7,"cloud_type":"Stratus"}
{"local_date_time":"30/02:00am","weather":"-","cloud":"Mostly clear","cloud_base_m":360,"cloud_oktas":1,"cloud_type_id":7,"cloud_type":"Stratus"}
{"local_date_time":"30/01:30am","weather":"-","cloud":"Mostly clear","cloud_base_m":840,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/01:00am","weather":"-","cloud":"Mostly clear","cloud_base_m":450,"cloud_oktas":1,"cloud_type_id":7,"cloud_type":"Stratus"}
{"local_date_time":"30/12:30am","weather":"-","cloud":"Mostly clear","cloud_base_m":840,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"30/12:00am","weather":"Recent precip.","cloud":"Mostly cloudy","cloud_base_m":600,"cloud_oktas":7,"cloud_type_id":38,"cloud_type":"-"}
{"local_date_time":"29/11:30pm","weather":"-","cloud":"Mostly clear","cloud_base_m":750,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"29/11:00pm","weather":"-","cloud":"Mostly clear","cloud_base_m":840,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"29/10:30pm","weather":"Thunder","cloud":"Mostly clear","cloud_base_m":360,"cloud_oktas":1,"cloud_type_id":7,"cloud_type":"Stratus"}
{"local_date_time":"29/10:15pm","weather":"Thunderstorm","cloud":"Mostly clear","cloud_base_m":360,"cloud_oktas":1,"cloud_type_id":7,"cloud_type":"Stratus"}
{"local_date_time":"29/10:00pm","weather":"-","cloud":"Mostly clear","cloud_base_m":360,"cloud_oktas":1,"cloud_type_id":7,"cloud_type":"Stratus"}
{"local_date_time":"29/09:53pm","weather":"Showers","cloud":"Mostly clear","cloud_base_m":450,"cloud_oktas":1,"cloud_type_id":7,"cloud_type":"Stratus"}
{"local_date_time":"29/09:30pm","weather":"Showers","cloud":"Mostly clear","cloud_base_m":300,"cloud_oktas":2,"cloud_type_id":7,"cloud_type":"Stratus"}
{"local_date_time":"29/09:00pm","weather":"Thunderstorm","cloud":"Mostly cloudy","cloud_base_m":300,"cloud_oktas":7,"cloud_type_id":39,"cloud_type":"-"}
{"local_date_time":"29/08:53pm","weather":"Thunderstorm","cloud":"Mostly clear","cloud_base_m":450,"cloud_oktas":1,"cloud_type_id":7,"cloud_type":"Stratus"}
{"local_date_time":"29/08:30pm","weather":"-","cloud":"Mostly clear","cloud_base_m":780,"cloud_oktas":2,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"29/08:00pm","weather":"-","cloud":"Mostly clear","cloud_base_m":750,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"29/07:30pm","weather":"-","cloud":"Mostly clear","cloud_base_m":750,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"29/07:00pm","weather":"-","cloud":"Mostly clear","cloud_base_m":750,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"29/06:30pm","weather":"-","cloud":"Mostly clear","cloud_base_m":750,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"29/06:00pm","weather":"Distant precip.","cloud":"Mostly clear","cloud_base_m":600,"cloud_oktas":2,"cloud_type_id":38,"cloud_type":"-"}
{"local_date_time":"29/05:30pm","weather":"-","cloud":"Mostly clear","cloud_base_m":750,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"29/05:00pm","weather":"-","cloud":"Mostly clear","cloud_base_m":900,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"29/04:30pm","weather":"-","cloud":"Mostly clear","cloud_base_m":750,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"29/04:00pm","weather":"-","cloud":"Mostly clear","cloud_base_m":900,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"29/03:30pm","weather":"-","cloud":"Mostly clear","cloud_base_m":900,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"29/03:00pm","weather":"Fine","cloud":"Mostly cloudy","cloud_base_m":600,"cloud_oktas":7,"cloud_type_id":38,"cloud_type":"-"}
{"local_date_time":"29/02:30pm","weather":"-","cloud":"Mostly clear","cloud_base_m":900,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"29/02:00pm","weather":"-","cloud":"Mostly clear","cloud_base_m":900,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"29/01:30pm","weather":"-","cloud":"Mostly clear","cloud_base_m":900,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"29/01:00pm","weather":"-","cloud":"Mostly clear","cloud_base_m":900,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"29/12:30pm","weather":"-","cloud":"Mostly clear","cloud_base_m":900,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"29/12:00pm","weather":"Fine","cloud":"Mostly cloudy","cloud_base_m":600,"cloud_oktas":6,"cloud_type_id":38,"cloud_type":"-"}
{"local_date_time":"29/11:30am","weather":"-","cloud":"Mostly clear","cloud_base_m":900,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"29/11:00am","weather":"-","cloud":"Mostly clear","cloud_base_m":900,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"29/10:36am","weather":"-","cloud":"Mostly clear","cloud_base_m":900,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"29/10:30am","weather":"-","cloud":"Mostly clear","cloud_base_m":900,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"29/10:00am","weather":"-","cloud":"Mostly clear","cloud_base_m":900,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"29/09:30am","weather":"-","cloud":"Mostly clear","cloud_base_m":750,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"29/09:00am","weather":"Fine","cloud":"Partly cloudy","cloud_base_m":600,"cloud_oktas":3,"cloud_type_id":38,"cloud_type":"-"}
{"local_date_time":"29/08:30am","weather":"-","cloud":"Mostly clear","cloud_base_m":750,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"29/08:00am","weather":"-","cloud":"Mostly clear","cloud_base_m":750,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
{"local_date_time":"29/07:30am","weather":"Showers","cloud":"Mostly clear","cloud_base_m":750,"cloud_oktas":1,"cloud_type_id":8,"cloud_type":"Cumulus"}
```